images/ebs-warmup/warmup-steps.sh

```diff
@@ -34,6 +34,12 @@ Supported flags:
 EOF
 }
 
+warmup_by_file() {
+    checkpoint=.com.pingcap.tidb.operator.ebs.warmup.checkpoint
+    /warmup --type=whole --files="$1" -P256 --direct --checkpoint.at="$1/$checkpoint"
```

Comment:
Are we sure we can warmup.sh has write privilege to tikv data volumes? @YuJuncen @WangLe1321